src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,387 @@
+sub-01/
+    microscopy/
+        sub-01_sample-01_stain-01_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-01_<modality_suffix>.json
+        sub-01_sample-01_stain-02_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-02_<modality_suffix>.json
+        sub-01_sample-01_stain-03_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-03_<modality_suffix>.json
+```
+In this example, the entity stain is used to distinguish images with different
+stains in separate files from the same sample.
+In the case where a single file contains different staining in each channel, the
+`stain-<label>` is omitted.
+
+Stains or antibodies SHOULD be indicated as appropriate in the `"SampleStaining"`,
+`"SamplePrimaryAntibodies"` and/or `"SampleSecondaryAntobodies"` keys in the sidecar JSON file,
+although the label may be different.
+
+If more than one run of the same sample, acquisition and stain are acquired during the same
+session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
+If only one run was acquired the `run-<index>` can be omitted.
+
+The [`chunk-<index>`](../99-appendices/09-entities.md#chunk) entity is used when multiples
+regions (2D images or 3D volumes files) of the same physical sample are imaged with different
+fields of view, regardless if they overlap or not.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+Four chunks (`chunk-01` to `chunk-04`) from the same brain sample (`sample-01`) of subject `sub-01`.
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_chunk-01_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-01_<modality_suffix>.json
+        sub-01_sample-01_chunk-02_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-02_<modality_suffix>.json
+        sub-01_sample-01_chunk-03_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-03_<modality_suffix>.json
+        sub-01_sample-01_chunk-04_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-04_<modality_suffix>.json
+```
+In this example, JSON metadata is different for each chunk of `sub-01_sample-01`.
+JSON metadata may be defined per sample or per chunk as appropriate, per the
+[inheritance principle](../02-common-principles.md#the-inheritance-principle).
+
+In some cases, the chunks can be "ordered" and correspond to the displacement of the
+microscope stage for example.
+In others cases, the chunks can be different images of the same sample with no explicit
+spatial relation between them.
+
+Examples of different chunks configurations can be seen in Figure 1.
+![Figure 1](images/microscopy_chunks.png "Examples of Microscopy chunks")
+
+Figure 1: Examples of chunks configurations.
+
+-   a) ordered 2D chunks without overlap,
+-   b) ordered 2D chunks with overlap,
+-   c) unordered 2D chunks with and without overlap,
+-   d) and e) ordered 2D chunks on different 3D planes,
+-   f) ordered 3D chunks.
+
```

Comment:
It would be very nice if this figure example, which is very clarifying, could be translated into an example with corresponding file names and specific fields, to have a demonstration of how this would be done in practical terms

Reply:
This may also relate to the comments in the Chunck transformations section

Reply:
Thank you for your feedback, I have a few questions about the examples you had in mind.

In each configuration case, the naming of the files is the same as the one exampled above with "n" number of chunks depending on the configuration. So I would not want to add another similar example.

My suggestion to avoid redundancy would be to present the figure first, and then the filenaming example. We could add `chunk-05` and `06` as an example of `Figure 1)b)` or `f)`.

As for the specific fields, an example of `Figure 1b)` of `f)` json metadata fields for 1 chunk could be added to he Chunk Tranformation section?

Let me know your thoughts and if I interpreted your comments correctly.
Thanks!

Reply:
Ok, maybe I have a more general question first, how do people know how the chuncks align? Just by looking at `ChunkTransformationMatrix` and `ChunkTransformationMatrixAxis`?
If this is the case, then maybe add this 2 fields for each of the cases in the example. Just a few chunks per case is fine to give an idea of how this should work.

Reply:
For my use case (slightly overlapping chunks), I've tried to keep the chunks in the order they appear in the Y axis, where they overlap, but I believe the intention is that the chunk numbers can be arbitrarily ordered and that the chunk transform matrix controls the order / positioning.

Reply:
> Ok, maybe I have a more general question first, how do people know how the chuncks align? Just by looking at `ChunkTransformationMatrix` and `ChunkTransformationMatrixAxis`?

> the intention is that the chunk numbers can be arbitrarily ordered and that the chunk transform matrix controls the order / positioning.

Yes, that is exact. The numbers themselves are arbitrary (just an index) and the positions of the chunks is determined by the matrix.

I'll make a few changes today in this section to clarify those important points, thank you!

Reply:
Thanks a lot! I think having chunks is one of the main differences with respect to other extensions. Let's try to clarify and add some examples to this part, so it's easy to follow. Having chunk metadata organized will help in data reusage and tool development. I would incline towards having these fields required in the presence of chunks (with the option of unordered or equivalent, to include all cases)

Reply:
Thank you @guiomar and @LeeKamentsky for your suggestions. I re-ordered the chunk-index section and added a paragraph to recommend the use of Chunk Transformations in the case of ordered chunks, with a link to the corresponding JSON metadata section.

Reply:
Nice! That is super-clear to me. Thanks, @mariehbourget.

Reply:
Thnaks @mariehbourget! I think it reads better now :)
Maybe no need to add the 6 chuncks data example. I'd be more interested in knowing how I should concatenate them in case I want to develop some tools to analysze the data. And I can't identify this information clearely in the text?

Reply:
Thank you!
In the paragraph under the filename example, we linked to the Chunk Tranformation section and recommend using the matrix.
```
The index number can be assigned arbitrarily and, in the case of "ordered" chunks, the chunks' positions relative to one another SHOULD be defined by an affine transformation matrix in the JSON sidecar file of each chunk, as described in Chunk Transformations.
```
If we want to add JSON example, I would do it in the Chunk Transformation section directly, to keep the metadata section together.